LICENSES/README.md

```diff
@@ -27,7 +27,5 @@ The following exceptions apply:
 * Files `selinux/meson.build` and `selinux/build-selinux.sh` are copied from
   [flatpak](https://github.com/flatpak/flatpak) project and they are licensed under the original license,
   **LGPL-2.1-or-later**.
-* Files under `src/libhirte/ini/` are copied from [inih](https://github.com/benhoyt/inih) project and they're
-  licensed under the original license, **BSD-3-Clause**.
-* Files under `src/libhirte/hashmap/` are copied from [hashmap.c](https://github.com/tidwall/hashmap.c) project
-  and they are licensed under the original license, **MIT**.
+* Files under [`subprojects/`](../subprojects/) are integrated as git submodules and they are licensed under the
+respective original license.
```

Comment:
I don't think it's enough, we need to mention clearly what licences are used for each subproject here

Reply:
I wanted to be lazy and avoid that, but if it has to be then its fine with me.

Reply:
Done :heavy_check_mark: